core/controllers/suggestion.py

```diff
@@ -174,3 +174,20 @@ def get(self):
 
         self.values.update({'suggestions': [s.to_dict() for s in suggestions]})
         self.render_json(self.values)
+
+
+class EditSuggestionHandler(base.BaseHandler):
+    """Handler for editing the submitted suggestion by reviwer before
+    accepting.
+    """
+
+    @acl_decorators.can_edit_suggestion
+    def put(self, suggestion_id):
+        """Edits the submitted suggestion by author or reviewer.
+
+        Args:
+            suggestion_id: str. Id of suggestion to be edit.
+        """
+        new_change = self.payload.get('change')
```

Comment:
Naming... This is a dict right? So it should be named new_change_dict.

Reply:
yes ok

Reply:
done